src/clj/game/core/runs.clj

```diff
@@ -313,41 +313,40 @@
         cant-pay {:prompt "You can't pay the cost to access this card"
                   :choices ["OK"]
                   :delayed-completion true
-                  :effect (effect (resolve-access-ends eid c))}]
+                  :effect (effect (access-end eid c))}]
     (cond
       ;; Check if a pre-access-card effect trashed the card (By Any Means)
       (not (get-card state c))
-      (resolve-access-ends state side eid c)
+      (access-end state side eid c)
 
       ;; Either there were no access costs, or the runner could pay them.
       (empty? acost)
-      (trigger-access-events state side eid c title)
+      (access-trigger-events state side eid c title)
 
       (not-empty acost)
       ;; Await the payment of the costs; if payment succeeded, proceed with access.
       (when-completed (pay-sync state side anon-card acost)
                       (if async-result
-                        (trigger-access-events state side eid c title)
+                        (access-trigger-events state side eid c title)
                         (resolve-ability state :runner eid cant-pay c nil)))
       :else
       ;; The runner cannot afford the cost to access the card
       (resolve-ability state :runner eid cant-pay c nil))))
 
-(defn handle-access
-  "Apply game rules for accessing the given list of cards (which generally only contains 1 card.)"
-  ([state side cards] (handle-access state side (make-eid state) cards nil))
-  ([state side eid cards] (handle-access state side eid cards (:title (first cards))))
-  ([state side eid cards title]
-   ;; Indicate that we are in the access step.
+(defn access-card
+  "Apply game rules for accessing the given card."
+  ([state side card] (access-card state side (make-eid state) card nil))
+  ([state side eid card] (access-card state side eid card (:title card)))
+  ([state side eid c title]
```

Comment:
If we're no longer doing multiple cards, can we just call it `card` instead of `c`?